### PR TITLE
Jump Search through Java

### PR DESCRIPTION
**Pull request by navendu29**

### CODE REVIEW

File: code/search/interpolation_search/Interpolation.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
+package arrays;
```
Comment (AdiChat) [inline code/search/interpolation_search/Interpolation.java]: This line must be removed as an end user may not be using the package name and thus, will result in error.

Removing it makes the code perfect.

File: code/search/jump_search/JumpSearch.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
+package arrays;
```
Comment (AdiChat) [inline code/search/jump_search/JumpSearch.java]: This line must be removed as an end user may not be using the package name and thus, will result in error.

Removing it makes the code perfect.

**Comment by navendu29** [comment]
Removed the unwanted line. You can merge the PR now. Thanks

**Comment by AdiChat** [comment]
Looks good 👍 

Hope you are enjoying your journey with **OpenGenus**

**Pull request merged**